readline-complete.el

```diff
@@ -135,10 +135,18 @@
 ;; editline instead of readline. Consider recompiling bash, python,
 ;; etc with readline support instead.
 
+;; *** Note: emacs contained a bug from ~24.3.91 to ~24.5-rc3-fixed where
```

Comment:
Your bug was reported for 25.0.50, so the bit about 24.5-rc3-fixed seems off to me? Or do you mean that is when the bug was introduced?